src/components/Dashboard/FilingHistoryList/HeaderActions.vue

```diff
@@ -73,15 +76,14 @@ export default class HeaderActions extends Vue {
   @Prop({ required: true }) readonly filing!: ApiFilingIF
   @Prop({ required: true }) readonly index!: number
 
-  @Getter isBenBcCccUlc!: boolean
-  @Getter isFirm!: boolean
-  @Getter isRoleStaff!: boolean
+  @Getter(useBusinessStore) isBenBcCccUlc!: boolean
+  @Getter(useBusinessStore) isFirm!: boolean
+  @Getter(useRootStore) isRoleStaff!: boolean
 
-  @Action showCommentDialog!: ActionBindingIF
-  @Action toggleFilingHistoryItem!: ActionBindingIF
-
-  @Mutation mutateCurrentFiling!: (x: ApiFilingIF) => void
-  @Mutation mutateFileCorrectionDialog!: (x: boolean) => void
+  @Action(useFilingHistoryListStore) showCommentDialog!: ActionBindingIF
+  @Action(useFilingHistoryListStore) toggleFilingHistoryItem!: ActionBindingIF
+  @Action(useFilingHistoryListStore) setCurrentFiling!: (x: ApiFilingIF) => void
+  @Action(useFilingHistoryListStore) setFileCorrectionDialog!: (x: boolean) => void
```

Comment:
What do you think about changing all actions to have this kind of "type", in a future ticket? I think it works better for typings, but of course it will be some work to change them all.

Reply:
I do agree with you. I really like this instead of having `ActionBindingIF` for everything. Shall I create a ticket for that?